code/tests/edge/test_implementationDetails.brs

```diff
@@ -0,0 +1,21 @@
+' ********************** Copyright 2023 Adobe. All rights reserved. **********************
+
+' This file is licensed to you under the Apache License, Version 2.0 (the "License");
+' you may not use this file except in compliance with the License. You may obtain a copy
+' of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+' Unless required by applicable law or agreed to in writing, software distributed under
+' the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+' OF ANY KIND, either express or implied. See the License for the specific language
+' governing permissions and limitations under the License.
+
+' *****************************************************************************************
+
+' target: _adb_ImplementationDetails()
+' @Test
+sub TC_adb_ImplementationDetails()
+    implementationDetails = _adb_ImplementationDetails()
+    UTF_assertEqual(implementationDetails["name"], "https://ns.adobe.com/experience/mobilesdk/roku")
+    UTF_assertEqual(implementationDetails["version"], "1.0.0-alpha1")
```

Comment:
update with version. nit: use test constant for all tests so that  we need to change only once.

Reply:
Haha, I'm not a supporter of using `test constant`. Let's discuss it after release.